plugin.video.catchuptvandmore/resources/lib/resolver_proxy.py

```diff
@@ -422,7 +508,8 @@ def get_francetv_video_stream(plugin,
         final_video_url = json_parser2['url']
         if download_mode:
             return download.download_video(final_video_url)
-        return final_video_url + '|User-Agent=' + web_utils.get_random_ua()
+        return final_video_url + '|X-Forwarded-For=' + '2.' + str(randint(0, 15)) + '.' + str(
```

Comment:
Why do you need the X-Forwarded-For?

Reply:
Once, I removed it. Some users complained France did not work anymore. So I added it again. But I always thought something was wrong in their system. You are right.  Better to remove it.

Reply:
I is usually used to circumvent Geo-IP protection. Of course, that is not something we want/allow.